rustler_tests/native/rustler_test/src/test_codegen.rs

```diff
@@ -225,6 +225,39 @@ pub fn tuplestruct_record_echo(tuplestruct: TupleStructRecord) -> TupleStructRec
     tuplestruct
 }
 
+mod check_if_only_decode_is_enough {
```

Comment:
There is also a test for [encode, decode](https://github.com/rusterlium/rustler/blob/6101dae30bc073f6af6971ac5565f24b6f814e74/rustler_tests/native/rustler_test/src/test_codegen.rs#L31) further above in this file, maybe that test should reference the `mod` here or so..